specifyweb/frontend/js_src/lib/components/InitialContext/treeRanks.ts

```diff
@@ -118,20 +104,50 @@ function getTreeScope(
 
 export function getTreeDefinitionItems<TREE_NAME extends AnyTree['tableName']>(
   tableName: TREE_NAME,
-  includeRoot: boolean
-): typeof treeDefinitions[TREE_NAME]['ranks'] | undefined {
-  const definition = caseInsensitiveHash(treeDefinitions, tableName);
-  return definition?.ranks.slice(includeRoot ? 0 : 1);
+  includeRoot: boolean,
+  rawDefinitionName?: string,
+  getAllRanks?: boolean
+): typeof treeDefinitions[TREE_NAME][number]['ranks'] | undefined {
```

Comment:
Type can be changed to `TreeInformationBackend[TREE_NAME][number]['ranks'] | undefined` ?